outputs.tf

```diff
@@ -5,10 +5,20 @@ output "container_app_environment_id" {
 
 output "container_app_fqdn" {
   description = "The FQDN of the Container App's ingress."
+  value       = { for name, container in azurerm_container_app.container_app : name => "${try(container.ingress[0].fqdn, "")}" if can(container.ingress[0].fqdn) }
```

Comment:
Hi @lonegunmanb 
The expression is just a copy from the URI output, I removed only the "https://" part. It will get the ingress FQDN from each container app and put it into a map with index of the container app name.

I'll send an update for the tests via squash commit.